dask_jobqueue/tests/test_scheduler_options_from_config.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+import psutil
+import dask
+
+from dask_jobqueue import (
+    PBSCluster,
+    MoabCluster,
+    SLURMCluster,
+    SGECluster,
+    LSFCluster,
+    OARCluster,
+)
+
+
+@pytest.mark.parametrize(
+    "Cluster",
+    [PBSCluster, MoabCluster, SLURMCluster, SGECluster, LSFCluster, OARCluster],
+)
+def test_import_scheduler_options_from_config(Cluster):
+
+    net_if_addrs = psutil.net_if_addrs()
+
+    config_scheduler_interface = list(net_if_addrs.keys())[0]
+    config_scheduler_port = 8804
+
+    pass_scheduler_interface = list(net_if_addrs.keys())[1]
+
+    generic_cluster_conf = {
```

Comment:
Do you need `generic_cluster_conf` i.e. can you not pass a smaller dict without these keys set to their default value?

Reply:
Thanks for questioning this approach, I now came up with a shorter implementation. I guess, though, the drawback is the very "hacky" retrieval of the cluster's `config_name`. Any idea on how to improve that?

By thinking about where to retrieve the `config_name` variable, I also stumpled across an inconsistency for the cluster implementation's `config_name` variables. As it would be good practice to do so, I have opened a separate PR on it. (Should probably also have done that for moving these `extra: []` entries in the configuration...)

Reply:
> Thanks for questioning this approach, I now came up with a shorter implementation. I guess, though, the drawback is the very "hacky" retrieval of the cluster's config_name. Any idea on how to improve that?

Well, I have just solved that (stupid) "hacky" part... still open for suggestions on the present approach, though. ;)